setup.py

```diff
@@ -59,7 +59,7 @@
         'pyusb>=1.0.0b2,<2.0',
         'pywinusb>=0.4.0;platform_system=="Windows"',
         'hidapi;platform_system=="Darwin"',
-        'pyyaml>=4.2b1,<5.0',
+        'pyyaml>=5.1,<6.0',
```

Comment:
Since this didn't require any code changes, you could leave the min version at 4.x, right? So its more compatible with other environments?

Reply:
D'oh, sorry I didn't see your review. No, the 4.x versions are "bad" according to the pyyaml authors. Seems they even plan to remove them from PyPI. See the info in #582.

Reply:
Oh good point, I hadn't quite got that far in the post! Thanks for the response.